probe/endpoint/ebpf.go

```diff
@@ -84,6 +96,22 @@ func isKernelSupported() error {
 		return fmt.Errorf("got kernel %s but need kernel >=4.4", release)
 	}
 
+	if strings.Contains(version, "Ubuntu") {
+		// Check for specific Ubuntu kernel versions with
+		// known issues.
+
+		abiNumber, err := strconv.Atoi(releaseParts[4])
+		if err != nil {
+			return err
```

Comment:
I think this should return nil, on the basis it's at least 4.4 and we don't know that it is 119-ish, so just allow it.

Reply:
Updated, thanks.